solutions/src/queries/getData.js

```diff
@@ -0,0 +1,14 @@
+// Add code below to query your database
+const dbConnection = require('../database/db_connection.js');
+
+const getData = (cb) => {
+    dbConnection.query(`SELECT * FROM users`, (err, res) => {
+        if {
+            (err) cb(err);
```

Comment:
Aha - thought I fixed this! The branching situation is confusing me. Will do it now